Github_News_Feed_Filter/Github_News_Feed_Filter.user.js

```diff
@@ -325,7 +325,7 @@
 		Array.prototype.map.call(newsContainer.querySelectorAll(':scope > div > .body'), function(alert) {
 			return alert.parentNode;
 		}).forEach(function(alert) {
-			var alertRepo = alert.querySelector(':scope .flex-items-baseline > div > [data-ga-click*="target:repo"]');
+			var alertRepo = alert.querySelector('[data-ga-click*="target:repo"]');
```

Comment:
I prefer to use the `:scope` selector

Reply:
But, when using `:scope .flex-items-baseline > div > [data-ga-click*="target:repo"]`
then there are some entries missing in the 'Repositories' tab, i.e. the global counter might be e.g. 30, but only 27 entries are listed, as I've wrote [here](https://github.com/jerone/UserScripts/issues/124#issuecomment-390481780).
Do you mean `:scope [data-ga-click*="target:repo"]` ?

Reply:
> Do you mean `:scope [data-ga-click*="target:repo"]`.

Yes (haven't tested it).

Reply:
OK, added it. It works ok as well.